sdcm/sct_config.py

```
@@ -3097,22 +3136,10 @@ def log_config(self):
-            if isinstance(data_nodes_num, list)
-            else [data_nodes_num]
-            if isinstance(data_nodes_num, int)
-            else [int(i) for i in str(data_nodes_num).split()]
-        )
+        zero_nodes_num: list[int] | int = self.get("n_db_zero_token_nodes")
+        data_nodes_num: list[int] | int = self.get("n_db_nodes")
+        zero_nodes_num = zero_nodes_num if isinstance(zero_nodes_num, list) else [zero_nodes_num]
+        data_nodes_num = data_nodes_num if isinstance(data_nodes_num, list) else [data_nodes_num]
         total_nodes = data_nodes_num[:]
         if use_zero_nodes and zero_nodes_num:
             total_nodes = [n1 + n2 for n1, n2 in zip(data_nodes_num, zero_nodes_num)]
```

Ruff (0.15.15)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)